context_engineering_functions.py

```diff
@@ -0,0 +1,91 @@
+import pandas as pd
+import os
+
+def get_available_maps(df):
```

Comment:
When I run this, I'm getting a lot of DecisionOrder = 1 with 0 maps available.  You can see it with this code:

Reply:
Were you just running this individually on the map_picks.csv? Just curious because Sid did fix a bug regarding this, and when I use in `create_basic_triples` I don't really see anything wrong....